scripts/rollup/packaging.js

```diff
@@ -165,15 +167,81 @@ function copyNodePackageTemplate(packageName) {
     // We already created this package (e.g. due to another entry point).
     return Promise.resolve();
   }
-  // TODO: verify that all copied files are either in the "files"
-  // whitelist or implicitly published by npm.
-  return asyncCopyTo(npmFrom, to).then(() =>
-    Promise.all([
-      asyncCopyTo(resolve(`${from}/package.json`), `${to}/package.json`),
-      asyncCopyTo(resolve(`${from}/README.md`), `${to}/README.md`),
-      asyncCopyTo(resolve('./LICENSE'), `${to}/LICENSE`),
-    ])
-  );
+  // Get all entry points in the package root
+  // Exceptions: *.fb.js
+  const packageEntries = glob.sync('!(*.fb).js', {
+    cwd: from,
+  });
+  const npmFiles = fs.readdirSync(npmFrom);
+  packageEntries.forEach(entry => {
+    if (!npmFiles.includes(entry)) {
+      // Terminate the build if any entry point(Exception: *.fb.js)
+      // does not have an equivalent in ./npm.
+      console.error(
+        `Entry point ${entry} in package ${packageName} does not have an equivalent in ./npm`
+      );
+      process.exit(1);
+    }
+  });
+  const packageJson = `${from}/package.json`;
+  const whitelist = fs.existsSync(packageJson) && require(packageJson).files;
+  let promisesForForwardingModules = [];
+  if (!whitelist) {
+    // Terminate the build if 'files' field is missing from package.json.
+    console.error(
+      `'files' field is missing from package.json in package ${packageName}`
+    );
+    process.exit(1);
+  } else {
```

Comment:
Since this is a hard exit, can you remove the `else` and just leave it after this code unindented?